src/storybook/changelog.mdx

```diff
@@ -16,6 +16,11 @@ We will publish release version history and changes here. Where possible, we wil
 
 Beta versions should be relatively stable but subject to occssional breaking changes.
 
+### **5.0.0-beta.15 - 03/10/24**
+
+- New component
+- `ButtonGroup` - Isolated the ButtonGroup from the button component to establish it as a new component, designed to accommodate a maximum of five button components as its children.
```

Comment:
**suggestion:** Probably worth mentioning that the old button group component has been deprecated and renamed to `DeprecatedButtonGroup`. Both of these changes constitute a breaking change too.

Reply:
@kurtdoherty Thank you for nitpick, have updated it now.